### DISCUSSION TITLE
Export to Web Ideas, Examples, Submissions

### DISCUSSION THREAD

**Discussion by openaudible** [openaudible/openaudible#794] [Show and tell]
If you've made a custom html layout for displaying your OpenAudible library, we'd love to see it.
We'd like to let users pick and choose from several different layouts.. If you submit a layout, we can include it in our app, assuming it uses free-to-distribute html/js libraries and is reasonably self-contained.

In case you're unaware, OpenAudible exports a .js file that includes all your books as variable, as well as a .json file that includes your books as a JSON array for ajax use. Either can be used to create a web page that shows a web page list of your books. 

Feel free to contribute any ideas, feature requests, etc.

**Comment by wyattj84** [comment]
I love this new feature, and have been reviewing the code. My only suggestion would be to add a very simple HTML5 audio player be able play each Audiobook on the page. Just a thought.

**Reply by openaudible** [reply]
Thanks! Hoping someone finds an example web app with player ability. A simple player would be unsatisfying without the ability to keep track of your position...

**Comment by Jumboshrimpboat** [comment]
Searching (unsuccessfully) for a way to convert AAX to MP3 for over a dozen years.  Just wanted to add this is the only product of the many I've tried that works right out of the box.
